packages/terra-functional-testing/lib/terra-dev-site/doc/asset-server-service/AssetServerService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+```
+
+## Configuration
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+            // Asset server service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
+
+A path to a webpack configuration file.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: The asset server service will check the current working directory for a `webpack.config.js` file.
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          config: './path/webpack.config.js'
+        }]
+    ],
+    // ...
+};
+```
+
+### index
+
+The asset index.
+
+Type: `string`
+
+Required: `false`
+
+Default: `index.html`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          index: 'index.html',
+        }]
+    ],
+    // ...
+};
+```
+
+### locale
+
+An optional locale that will be passed into the webpack configuration as an environment variable.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          locale: 'en-US',
+        }]
+    ],
+    // ...
+};
+```
+
+### port
+
+The port the service will be hosted on.
+
+Type: `number`
+
+Required: `false`
+
+Default: `8080`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          port: 8080,
+        }]
+    ],
+    // ...
+};
+```
+
+### site
+
+A relative path to a directory of static assets. If provided, an express server will be launched to serve the directory assets.
+
+Note: The `site` configuration option will take precedence over `config` if both are provided.
+
+Type: `string`
+
+Required: `false`
+
+Default: `undefined`
+
+Example:
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+          site: './build',
+        }]
+    ],
+    // ...
+};
+```
+
+### theme
```

Comment:
do we want to group theme and locale as wepback options? since they won't apply to a static site?

Reply:
I could go either way on this one. `webpackOptions` as an option won't apply to the static site either, it's a little awkward either way. We could potentially name them `webpackTheme` and `webpackLocale` or just add a note here they only apply to webpack.

Reply:
At a minimum, add a note that these options only apply to webpack builds. I'm not stuck on any option.